ui/tests/integration/components/replication-secondary-card-test.js

```diff
@@ -0,0 +1,99 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+const REPLICATION_DETAILS = {
+  state: 'stream-wals',
+  connection_state: 'ready',
+  lastWAL: 0,
+  lastRemoteWAL: 10,
+  delta: 10,
+};
+
+const STATE_ERROR = {
+  state: 'idle',
+  connection_state: 'ready',
+  lastWAL: 0,
+  lastRemoteWAL: 10,
+  delta: 10,
+};
+
+const CONNECTION_ERROR = {
+  state: 'stream-wals',
+  connection_state: 'transient_failure',
+  lastWAL: 0,
+  lastRemoteWAL: 10,
+  delta: 10,
+};
+
+module('Integration | Enterprise | Component | replication-secondary-card', function(hooks) {
+  setupRenderingTest(hooks);
+  const title = 'States';
+  const hasErrorClass = false;
+
+  hooks.beforeEach(function() {
+    this.set('replicationDetails', REPLICATION_DETAILS);
+    this.set('stateError', STATE_ERROR);
```

Comment:
this line should be moved to above line 81 since that is the only test context which uses this variable. line 38 should be moved too. the `beforeEach` hooks here should only set variables that are going to be used in every single test (or at least the majority of them) -- that way it's easy to scan through the tests and see how the component is rendered differently according to each test ✨